src/dao/AppDAO.js

```diff
@@ -151,7 +155,19 @@ class AppDAO extends AbstractContractDAO {
     };
 
     getLOCs = (account: string) => {
```

Comment:
@Pavel-DV, why? The name of the class should correspond to it's role. 'AppDAO' name is telling us, that we should store there only some common functionality. Probably we don't need such class at all, but for sure we can and we should (and this is what actually you offered in our common correspondence in slack :)) extract LOCs logic out of here